drltrace_src/drltrace.cpp

```diff
@@ -300,78 +335,94 @@ lib_entry(void *wrapcxt, INOUT void **user_data)
     bool allowed = false;
     bool tested = false;  /* True only if any white/blacklist testing below is done. */
     for (unsigned int i = 0; (allowed == false) && (i < filter_function_whitelist_len); i++) {
-      tested = true;
-
-      /* If the whitelist entry contains a wildcard, then compare only the shortest
-       * part of either string. */
-      unsigned int module_name_len_compare;
-      if (filter_function_whitelist[i].is_wildcard)
-        module_name_len_compare = MIN(module_name_len, \
-          filter_function_whitelist[i].func_name_len);
-      else
-        module_name_len_compare = module_name_len;
-
-      if (fast_strcmp(module_name, module_name_len_compare, \
-          filter_function_whitelist[i].func_name, \
-          filter_function_whitelist[i].func_name_len) == 0) {
-        allowed = true;
-      }
-    }
-
-    /* Check the blacklist if it was specified instead of a whitelist. */
-    if (!allowed && filter_function_blacklist_len > 0) {
-      allowed = true;
-      for (unsigned int i = 0; allowed && (i < filter_function_blacklist_len); i++) {
         tested = true;
 
-	/* If the blacklist entry contains a wildcard, then compare only the shortest
-	 * part of either string. */
+        /* If the whitelist entry contains a wildcard, then compare only the shortest
```

Comment:
Are you sure you are using the latest code for this commit? I see a bunch of code removed for some reason...

Reply:
Yes, my master was up to date with your master.
It may be just due to the re-format.